docs/install/upgrade-from-mongodb.md

```diff
@@ -168,5 +164,5 @@ To upgrade a replica set or a sharded cluster, use the [rolling restart](../glos
 
 ## Upgrading to Percona Server for MongoDB with data at rest encryption enabled
 
-Steps to upgrade from MongoDB 6.0 Community Edition with data encryption enabled to Percona Server for MongoDB are different. `mongod` requires an empty `dbPath` data directory because it cannot encrypt data files in place. It must receive data from other replica set members during the initial sync. Please refer to the [Switching storage engines](../inmemory.md#switching-storage-engines) for more information on migration of encrypted data. [Contact us](https://www.percona.com/about-percona/contact#us) for working at the detailed migration steps, if further assistance is needed.
+Steps to upgrade from MongoDB 7.0 Community Edition with data encryption enabled to Percona Server for MongoDB are different. `mongod` requires an empty `dbPath` data directory because it cannot encrypt data files in place. It must receive data from other replica set members during the initial sync. Please refer to the [Switching storage engines](../inmemory.md#switching-storage-engines) for more information on migration of encrypted data. [Contact us](https://www.percona.com/about-percona/contact#us) for working at the detailed migration steps, if further assistance is needed.
```

Comment:
```suggestion
Steps to upgrade from MongoDB 7.0 Community Edition with data encryption enabled to Percona Server for MongoDB are different. `mongod` requires an empty `dbPath` data directory because it cannot encrypt data files in place. It must receive data from other replica set members during the initial sync. Please have a look at the [Switching storage engines](../inmemory.md#switching-storage-engines) for more information on the migration of encrypted data. [Contact us](https://www.percona.com/about-percona/contact#us) to work on the detailed migration steps if you need more help.
```